src/cpp/src/continuous_batching/model_runner.hpp

```diff
@@ -406,6 +520,23 @@ class ModelRunner {
 
         _reset_cache_rotation_coefficients();
 
+        if (_is_hs_export()) {
```

Comment:
it looks like I didn't fully understand the idea, could you please help with understanding ?
draft model should get the data from hidden state of target model - am i right ? what is hidden state include ? 
according to schema from CVS-170727 i thought that inputs for the draft model will be taken from some layers of target, but the draft model receives input data based on logits, how do data from hidden states participate in token generation?